### PR TITLE
Add capacity line to overall incarcerated population chart

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

Add a lien to the population projection chart indicating the capacity of the system. Currently only displays when viewing the total incarcerated population.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes [recidiviz-data/#6748]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/PopulationTimeSeriesChart/index.tsx [hunk 1]
```diff
@@ -164,6 +164,31 @@ const PopulationTimeSeriesChart: React.FC<PropTypes> = ({ data }) => {
...
+                : 1e6,
+            // Need to send this line off of the chart when not looking at all
+            // incarcerated people. We need to move it instead of deleting it
+            // to prevent semiotic from deleting and rerendering the annotation
+            // layer which makes the uncertainty band just appear at its new location
+            // instead of transforming its way there
+            disable: "connector",
+            color: styles.crimsonDark50,
+            note: {
+              label: "Total Operational Capacity (includes CAPP): 8,008",
```
Comment (daschi) [inline src/core/PopulationTimeSeriesChart/index.tsx:183]: Will this eventually come from the file's metadata? Maybe this should be updated to use a variable instead ?

**Review by daschi — APPROVED** [review]
Just a minor suggestion which I'll leave up to you

File: src/core/CoreConstants.scss [hunk 2]
```diff
@@ -15,6 +15,8 @@ $inset-shadow-30: inset 0px -1px 1px $box-shadow-color-30;
 $white: #fff;
 $crimson: #bf7474;
 $crimson-dark: #a43939;
+$crimson-dark-50: rgba(164, 57, 57, 0.5);
+$new-color: rgba(164, 57, 57, 0.5);
```
Comment (daschi) [inline src/core/CoreConstants.scss:19]: is this leftover?

File: src/core/PopulationTimeSeriesChart/index.tsx (hunk 1 above)
Comment (Catacola) [reply src/core/PopulationTimeSeriesChart/index.tsx:183]: We will eventually have this on a per-facility basis and maybe a per-population basis. I can update to be more template-like.

File: src/core/CoreConstants.scss (hunk 2 above)
Comment (Catacola) [reply src/core/CoreConstants.scss:19]: Yup!

**Pull request merged**